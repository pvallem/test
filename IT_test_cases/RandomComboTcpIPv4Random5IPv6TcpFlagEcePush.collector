bidir_flow_info {
  key {
    src_address: "\025@8L"
    dst_address: "<>\254\214"
    src_port: 17515
    dst_port: 35626
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 10
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "<\341\354\232"
    dst_address: "`:k\030"
    src_port: 1174
    dst_port: 6970
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\005\006\007\010\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "X\2517}"
    dst_address: "\257<\215\007"
    src_port: 13992
    dst_port: 30158
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200000040
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
