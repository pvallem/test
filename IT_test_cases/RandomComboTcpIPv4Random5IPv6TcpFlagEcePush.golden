flow_info {
  key {
    src_address: "<\341\354\232"
    dst_address: "`:k\030"
    src_port: 1174
    dst_port: 6970
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\001\002\003\004\000\000\000\000\000\000\000\000\000\000\000\000"
    dst_address: "\005\006\007\010\000\000\000\000\000\000\000\000\000\000\000\000"
    src_port: 111
    dst_port: 222
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200000030
}
flow_info {
  key {
    src_address: "\025@8L"
    dst_address: "<>\254\214"
    src_port: 17515
    dst_port: 35626
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: IPV4
  }
  start_time: 1388563200000020
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 10
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\257<\215\007"
    dst_address: "X\2517}"
    src_port: 30158
    dst_port: 13992
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000040
    key_type: IPV4
  }
  start_time: 1388563200000040
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000040
}
